sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -37,6 +37,20 @@
 
 #define _NETIF_NETAPI_MSG_QUEUE_SIZE    (8)
 
+#ifdef MODULE_GNRC_NETIF_EVENTS
+/**
+ * @brief   Event type used for passing netdev pointers together with the event
+ */
+typedef struct {
+    event_t super;
+    netdev_t *dev;
+} event_netdev_t;
+
+typedef struct {
+    event_netdev_t isr;
+} gnrc_netif_events_t;
```

Comment:
Why are those split up into into two types?

Reply:
Originally I had thought that there would be more than isr in this struct, but it showed that it was not necessary. I have removed this now and refactored the code.